### PR TITLE
OD-493 [Feature] New messages format using file picker

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-493 https://weboo.atlassian.net/browse/OD-493
## Description
Added new messages formats using file picker
## Screenshots/screencasts
https://screencast-o-matic.com/watch/crfOI1VerHt
## Backward compatibility
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @ivandevupp @YaroslavOvdii

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -376,6 +386,37 @@ function getOrganizations() {
     });
 }
 
+function deleteConfirmation(type, id) {
```
Comment (squallstar) [inline js/interface.js:389]: it sounds like this function should be called "displayDeletionConfirmation"

File: js/interface.js [hunk 2]
```diff
@@ -376,6 +386,37 @@ function getOrganizations() {
...
+  var sourceType = type === 'folder'
+    ? 'folders'
+    : 'files';
+
+  return new Promise(function(resolve, reject) {
```
Comment (squallstar) [inline js/interface.js:394]: I believe `Fliplet.Modal.confirm` already returns a promise, why did you wrap this into a new Promise?

File: js/interface.js [hunk 3]
```diff
@@ -595,6 +636,48 @@ function restoreRoot(appId, organizationId) {
...
+          className: 'btn-primary'
+        }
+      }
+    }).then(function(result) {
+      if (result) {
+        return;
+      }
+
+      var navStack = {};
+      navStack.tempStack = cleanNavStack();
```
Comment (squallstar) [inline js/interface.js:672]: empty line after var declaration

File: js/interface.js [hunk 4]
```diff
@@ -595,6 +636,48 @@ function restoreRoot(appId, organizationId) {
...
+      navStack.upTo = cleanNavStack();
+      navStack.upTo.pop(); // Remove last one
+
+      initFileManagerOverlay(navStack);
+    });
+  })
```
Comment (squallstar) [inline js/interface.js:678]: missing semicolon

**Review by squallstar — CHANGES_REQUESTED** [review]
Rejected +1

**Comment by AndrRyaz** [comment]
@squallstar all comment are fixed

File: js/interface.js (hunk 4 above)
Comment (AndrRyaz) [reply js/interface.js]: fixed

File: js/interface.js (hunk 3 above)
Comment (AndrRyaz) [reply js/interface.js:669]: fixed

File: js/interface.js (hunk 2 above)
Comment (AndrRyaz) [reply js/interface.js]: your right, fixed

File: js/interface.js (hunk 1 above)
Comment (AndrRyaz) [reply js/interface.js]: changed the name

**Pull request merged**